### PR TITLE
Moved global overrides to development values file

### PR DESCRIPTION
**Pull request by claudiol**
- Moved global overrides to application section

### CODE REVIEW

**Comment by mbaldessari** [comment]
You left some personal info in values-global.yaml ;)

File: tests/common-clustergroup-industrial-edge-factory.expected.yaml [hunk 1]
```diff
@@ -135,18 +135,34 @@ data:
...
+        zapImage: quay.io/hybridcloudpatterns/zap2docker-stable
       cicd:
         namespace: devsecops-ci
       clusterDomain: region.example.com
       git:
-        account: PLAINTEXT
+        account: claudiol
         dev_revision: main
-        email: SOMEWHERE@EXAMPLE.COM
+        email: claudiol@redhat.com
```
Comment (beekhof) [inline tests/common-clustergroup-industrial-edge-factory.expected.yaml:155]: Probably didn't mean to put your details here right?

File: values-development.yaml [hunk 2]
```diff
@@ -85,6 +55,15 @@ clusterGroup:
...
         - /secrets
+      overrides:
+      - name: devel.image
+        value: chat-client:latest
+      - name: devel.npmbase
+        value: quay.io/hybridcloudpatterns/ubi-npm
+      - name: devel.namespace
+        value: devsecops-ci
+      - name: devel.roxCTLImage
+        value: registry.access.redhat.com/advanced-cluster-security/rhacs-roxctl-rhel8:3.71.0-5
```
Comment (beekhof) [inline values-development.yaml:66]: Is this mechanism preferred over the below?

```
devel: 
   image: chat-client:latest
   npmbase: quay.io/hybridcloudpatterns/ubi-npm
   namespace: devsecops-ci
   roxCTLImage: registry.access.redhat.com/advanced-cluster-security/rhacs-roxctl-rhel8:3.71.0-5
```

/cc @mhjacks @mbaldessari

Comment (beekhof) [reply values-development.yaml:66]: Obviously both will work, but we likely want to pick one consistently across the patterns

**Comment by claudiol** [comment]
Talked to Jonny about this.  The latest PR changes have been merged and we will need to figure out what our path forward is.

**Pull request closed without merging**